config/environments/development.rb

```diff
@@ -54,4 +57,10 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
Add `RAILS_LOG_TO_STDOUT` to your `.env` to log the output to standard out instead of the log file. I personally find this more convenient. Took this from the production config.